apps/settings/elements/firefox_sync.html

```diff
@@ -50,6 +50,12 @@ <h2 data-l10n-id="fxsync-collections">Collections</h2>
       </header>
       <ul role="presentation">
         <li role="presentation">
+          <gaia-switch class="fxsync-collections-bookmarks"
+                       name="sync.collections.bookmarks.enabled">
+            <label data-l10n-id="fxsync-collections-bookmarks">Bookmarks</label>
```

Comment:
We are not putting string into HTML any more (just the l10n-id data attr). Could you remove them from this template.